### PR TITLE
Focus README.md on swap development mode

### PR DESCRIPTION
**Pull request by pchico83**
Also, add a more visual explanation for the concept of developing in a container

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,95 +13,60 @@
...
+Okteto decouples deployment from development. You can deploy your application with kubectl, Helm, a serverless framework, or even a CI pipeline and use Okteto to develop it. This is especially useful for cloud-native applications where deployment pipelines are not trivial.
 
-Windows
-```
-Download https://downloads.okteto.com/cli/okteto.exe and add it to your $PATH.
-```
+Okteto replaces a running deployment by a development container. You write code from your local IDE and Okteto instantly synchronizes your code changes to your development container. The development container can be any docker image, where you install all the dev tools needed by your application: compilers, debuggers, hot reloaders... Okteto gives you a terminal to your development container to build, test, and run your application as you would from a local terminal. Files are synchronized both ways.
 
-You can also directly download the precompiled binary from [the releases page](https://github.com/okteto/okteto/releases).
+![Okteto](docs/okteto-architecture.png)
```
Comment (rlamana) [inline README.md:26]: maybe I can update the diagram to use the concepts of "development container" in it, and make some other improvements

File: README.md [hunk 2]
```diff
@@ -13,95 +13,60 @@
...
-
-### Instant file synchronization
-
-Okteto detects your code changes and synchronizes your code to your development environment (powered by [Syncthing](https://github.com/syncthing/syncthing)). Files are synchronized both ways. If you edit a file directly in your development environment, the changes will be reflected locally as well. Great for keeping your `package-lock.json` or `requirements.txt` up to date.
-
-### Keep Your Own Tools
+## Getting started
 
-Okteto easily integrates and augments your existing tools. You write code locally, which means Okteto is compatible with any IDE. And you can configure your own development environment runtime using a docker image, with your own compilers, linters, debuggers or hot reloaders.
+All you need to get started is to [install the Okteto CLI](https://okteto.com/docs/getting-started/installation/index.html) and have access to a Kubernetes cluster. You can also use Okteto in [Okteto Cloud](https://okteto.com/), the best development platform for Kubernetes applications.
```
Comment (rlamana) [inline README.md:37]: `the best development platform for Kubernetes applications`, I know this was already there but it still sounds odd to me. Maybe something like `our hosted development platform for Kubernetes applications` or `our approach to a hosted development platform for Kubernetes applications on top of Okteto`.

Comment (rberrelleza) [reply README.md:37]: I think it's better to remove that last sentence.

File: README.md (hunk 1 above)
Comment (pchico83) [reply README.md:34]: That would be cool. I am going to merge it, but if you can, update the image later.

**Pull request merged**